subroutines/src/pcs/univariate_kzg/mod.rs

```diff
@@ -110,7 +106,7 @@ impl<E: PairingEngine> PolynomialCommitmentScheme<E> for UnivariateKzgPCS<E> {
         let (num_leading_zeros, plain_coeffs) = skip_leading_zeros_and_convert_to_bigints(poly);
 
         let msm_time = start_timer!(|| "MSM to compute commitment to plaintext poly");
-        let commitment = VariableBaseMSM::multi_scalar_mul(
+        let commitment = E::G1::msm_bigint(
```

Comment:
What's the difference between `msm_bigint` and `msm_unchecked`?

Reply:
Basically, 
- `msm_unchecked` takes a ScalarField slice
- `msm_bigint` - a BigInt slice 

as the second argument. See their definitions at https://github.com/arkworks-rs/algebra/blob/v0.4.0/ec/src/scalar_mul/variable_base/mod.rs :

```rust
fn msm_unchecked(bases: &[Self::MulBase], scalars: &[Self::ScalarField])

fn msm_bigint(bases: &[Self::MulBase], bigints: &[<Self::ScalarField as PrimeField>::BigInt])
```

Reply:
Answering your question made me think if I could make it better, and indeed I can. 

Using `msm_unchecked` instead of `msm_bigint` allows to delete the conversion code (by letting the library take care of it).

This gives a sizable

Reply:
Fixed it.